fluent-react/src/markup.js

```diff
@@ -1,6 +1,12 @@
-/* eslint-env browser */
-
-const TEMPLATE = document.createElement("template");
+/* global window, require */
+let doc = null;
```

Comment:
Nit: I'd prefer to use a `var` in both branches of the `if…else`, but this is of course fine as well.